### DISCUSSION TITLE
Parent and child prefix

### DISCUSSION THREAD

**Discussion by 22Jean-Paul** [netbox-community/netbox#12668] [Q&A]
Hi Netbox community,

How can I do parent and child prefix.
Parent subnet is 192.168.0.0/22
child subnet is 192.168.50.0/25.

Thank's for your help !

**Comment by candlerb** [comment]
You simply create the two prefixes, 192.168.0.0/22 and 192.168.50.0/25. The parent-child relationship is implicit: when you browse 192.168.0.0/22 you'll see a "child prefixes" tab, and 192.168.50.0/25 will be under there.

Note that there is a prefix status field, and there's slightly different behaviour depending on whether you select "Container" for the outer one.  In particular, this affects the calculation of prefix utilization.

**Reply by markkuleinio** [reply]
What he said, plus:

```
>>> list(ipaddress.ip_network("192.168.0.0/22").subnets(new_prefix=25))
[IPv4Network('192.168.0.0/25'), IPv4Network('192.168.0.128/25'),
IPv4Network('192.168.1.0/25'), IPv4Network('192.168.1.128/25'),
IPv4Network('192.168.2.0/25'), IPv4Network('192.168.2.128/25'),
IPv4Network('192.168.3.0/25'), IPv4Network('192.168.3.128/25')]
```

= 192.168.50.0/25 is not within 192.168.0.0/22

**Reply by candlerb** [reply]
> = 192.168.50.0/25 is not within 192.168.0.0/22

D'oh, I didn't even occur to me to check that! Thank you.

Parent needs to be 192.168.0.0/18 (or /17 or /16) to have 192.168.50 as a child.
